cluster/store/store.go

```diff
@@ -332,6 +336,19 @@ func (st *Store) onLeaderFound(timeout time.Duration) {
 	}
 }
 
+// RestoreSchemaToV0() is responsible for saving new schema (RAFT) to boltDB
+func (st *Store) RestoreSchemaToV0() error {
+	cs := map[string]ClassState{}
+	for _, c := range st.db.Schema.Classes {
```

Comment:
To prevent race condition, I suggest delegating getting `cs` to the db.schema

Reply:
I am not sure that i get `delegating getting cs to the db.schema`, can you elaborate more ?

Reply:
Sure. we should make getting the schema a method `func (s *schema) LoadSchemaV1()` in schema.go
,as it can handle concurrent access